assets/js/util.js

```diff
@@ -119,6 +119,25 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Comment:
While removing the class, I remove `hidden-xs hidden-sm **hidden-md hidden-lg**` (basically all the hidden classes as we have to make them visible) and while adding the class, I add `hidden-xs hidden-sm  **visible-md visible-lg** `. The `hidden-xs hidden-sm` classes stay constant as this feature should not be rendered on mobile devices. Thus, when the scroll passes the height, I add the `visible-md visible-lg` classes to ensure they are visible only in `*-md *-lg` screens and are hidden in `*-sm *-xs` devices.